### PR TITLE
Firebase auth - Wip

### PR DESCRIPTION
**Pull request by tomheller**

### CODE REVIEW

File: src/components/atoms/login/Login.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    username() {
+      return this.loggedIn ? this.user.displayName : '';
+    },
+  },
+  methods: {
+    login() {
+      Service.login();
+    },
+    logout() {
+      console.log(Service);
```
Comment (ffriedl89) [inline src/components/atoms/login/Login.vue]: could be removed

**Review by ffriedl89 — APPROVED** [review]
looks really good, user spaces and all that 👍 🦑

**Comment by ffriedl89** [comment]
ready to merge?

**Comment by tomheller** [comment]
I'm currently not certain where and when we should pull the entries from the database. Maybe the app.vue should request the entries for the currently displayed week. To prevent duplicate queries, maybe the store should keep a record of which week entries have been already requested... 
idk yet, input is welcome. of course.

**Pull request merged**